### ISSUE TITLE
Brackets has several issues in Ubuntu 16.04 Xenial and Linux Mint Sarah (Updated with Fixes)

### ISSUE THREAD

**Issue by NathanJPlummer** [adobe/brackets#12514]
---
Updated Post
---

These issues are currently occurring on Brackets 1.7, installed from the PPA, on Linux Mint Sarah, based on Ubuntu 16.04.  The same issues were occurring a few days ago in vanilla Ubuntu.

- Experimental Live Preview not recognizing the index.html file.

![Imgur](http://i.imgur.com/h780mCi.jpg)

- Unable to install extensions

![Imgur](http://i.imgur.com/8ncUyJn.jpg)

- Attempting to enable node.js gives an error message.

![Imgur](http://i.imgur.com/JkzRJkY.jpg)

---
Original Post
---

After upgrading from Wily and Xenial, Brackets 1.6 fails to load the Experimental Live Preview.

Brackets 1.7 is showing the same issue (plus a few others already mentioned).  Here's a few different combinations I have tried while troubleshooting:

1.  Brackets 1.6 from PPA with clean install of Xenial.

2.  Brackets 1.7 from deb after deleting Brackets config

3.  Brackets 1.7 deb on a Linux Mint live CD (also based on Xenial)

No luck with any of these combinations, unfortunately.

---
Similar Issues
---

More issues with Xenial were reported here: #12515 

@onebelarusianguy

> I'm trying move from Windows(where is Brackets works well) to Ubuntu and it's make me sad :(
>  1. Remembering the last open state of project not working, as well as not working adding opened files to left and right panels in sidebar.
>  2. Double-click to close Brackets? wtf?
>  And I assume it's not all problems that can be on linux.
> 
> I love Brackets so much, please don't ignore linux users.
> 
> Builds that i've tried: 1.6, 1.7 - the same troubles
> I'm trying move from Windows(where is Brackets works well) to Ubuntu and it's make me sad :(
>  1. Remembering the last open state of project not working, as well as not working adding opened files to left and right panels in sidebar.
>  2. Double-click to close Brackets? wtf?
>  And I assume it's not all problems that can be on linux.
> 
> I love Brackets so much, please don't ignore linux users.
> 
> Builds that i've tried: 1.6, 1.7 - the same troubles

**Comment by NathanJPlummer** [comment]
#12515

More issues with Xenial were reported here,

**Comment by cakedoer** [comment]
Same problems with Brackets 1.6 and 1.7 on Arch Linux.

**Comment by NathanJPlummer** [comment]
I have fixed this issue.  I noticed one of the developers mention in another issue that the Linux binary has been refreshed  due to some issues.  I reinstalled the binary from the Brackets site, and viola! it worked.

I'm leaving this issue open since there's no version change indicating anythings changed and I really had to hunt for this information.  Also, I suspect for Ubuntu/Mint users many use the PPA since libcrypt11 isn't available in the default repository.

To get Brackets 1.7 working on Ubuntu 16.04 I followed the following steps:

- Installed libcrypt11 from [here](https://launchpad.net/~ubuntu-security/+archive/ubuntu/ppa/+build/8993248)
- For the 32-bit version of libcrypt11 go [here](https://launchpad.net/~ubuntu-security/+archive/ubuntu/ppa/+build/8993251)

- Install brackets from the website, NOT the PPA.

- Disable the PPA if you have it enabled to prevent automatic upgrades.
